docs/writing-helmfile.md

```diff
@@ -104,6 +104,8 @@ releases:
   version: 0.3.2
   inherit:
   - template: default
+    exclude:
```

Comment:
```suggestion
    except:
```

Nit!

Reply:
@voron I couldn't apply this suggestion by myself perhaps due to your repo's access control. It would be nice if you could just apply it so that I can approve this for merging.

Thanks in advance your contribution and support @voron!

Reply:
Fixed @mumoshu, thanks!